### ISSUE TITLE
IO: no warning is thrown if a file does not exist

### ISSUE THREAD

**Issue by SpiritSuperUser** [spirit-code/spirit#413]
Reading in a file which does not exist does not throw a warning.
Even worse: The API message in the Log-file suggests that the file was read...

**Comment by GPMueller** [comment]
Which version of the code?
And please attach a corresponding Log

**Comment by GPMueller** [comment]
I believe since PR #410 was merged with 4ae862a374f6b2be7d62bfc2c065397962af5fa5, problems like this should not be occurring anymore, instead a Log message like this should appear
```
2018-06-12 21:19:45  [WARNING] [API ] [01] [01]  IO_Chain_Read: file "nonexistent.ovf" does not seem to exist
```
